src/Microsoft.ML.Data/Transforms/ValueToKeyMappingTransformerImpl.cs

```diff
@@ -219,7 +219,7 @@ public override void ParseAddTermArg(ref ReadOnlyMemory<char> terms, IChannel ch
                 }
 
                 if (Count == 0)
-                    throw ch.ExceptUserArg(nameof(Arguments.Terms), "Nothing parsed as '{0}'", ItemType);
+                    throw ch.ExceptUserArg(nameof(Arguments.Term), "Nothing parsed as '{0}'", ItemType);
```

Comment:
Since you are chaning these strings, can you interpolate them?

Reply:
Done